Parse email files time out (#16186)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_6_5.md

#### Scripts
##### ParseEmailFiles

> - Fixed an issue where the attachment name was in an unknown encoding.

Reviewer: ```suggestion
- Fixed an issue where the encoding of an attachment name was mishandled
```
